### PR TITLE
feat(datastore): adds query filter OR

### PR DESCRIPTION
**Pull request by telpirion**
This sample adds the following region tags:

+ datastore_query_filter_or

### CODE REVIEW

**Review by kweinmeister — APPROVED** [review]

File: datastore/cloud-client/query_filter_or.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START datastore_query_filter_or]
+from google.cloud import datastore
+from google.cloud.datastore import query
+
+"""
+Builds a union (OR) query filter.
```
Comment (nicain) [inline datastore/cloud-client/query_filter_or.py:20]: Move outside region tags, single whitespace after license header and two whitespace before START tag

File: datastore/cloud-client/query_filter_or.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            query.PropertyFilter("description", "=", "Feed cats"),
+        ]
+    )
+
+    or_query.add_filter(filter=or_filter)
+
+    results = or_query.fetch()
+    for result in results:
+        print(result["description"])
+
```
Comment (nicain) [inline datastore/cloud-client/query_filter_or.py:48]: nit: remove whitespace

**Review by nicain — CHANGES_REQUESTED** [review]

File: datastore/cloud-client/query_filter_or_test.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    yield entities
+
+    client.delete(task1)
+    client.delete(task2)
+
+
+def test_query_filter_or(capsys, entities):
+    query_filter_or(project_id=PROJECT_ID)
+    out, _ = capsys.readouterr()
+    assert "Feed cats" in out
```
Comment (nicain) [inline datastore/cloud-client/query_filter_or_test.py:48]: nit: end in newline

File: datastore/cloud-client/query_filter_or.py (hunk 2 above)
Comment (telpirion) [reply datastore/cloud-client/query_filter_or.py]: Done.

File: datastore/cloud-client/query_filter_or.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+"""
+
+
+def query_filter_or(project_id: str) -> None:
+    """Builds a union of two queries (OR) filter.
+
+    Arguments:
+        project_id: your Google Cloud Project ID
+    """
+
```
[the remark this answers is not in the document]
Comment (telpirion) [reply datastore/cloud-client/query_filter_or.py]: Done.

**Review by nicain — APPROVED** [review]

**Pull request merged**